topics/cassandra/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: apache-cassandra
+created_by: Apache Software Foundation
+display_name: Apache Cassandra
+github_url: https://github.com/apache/cassandra
+logo: cassandra.png
+related: language, dotnet
+released: July 2008
+short_description: Apache Cassandra is a free, open source, distributed, NoSQL database management system.
+topic: cassandra
+url: http://cassandra.apache.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Apache_Cassandra
+---
+Apache Cassandra is a free and open source, distributed, wide column store, NoSQL database management system designed to handle large amounts of data across many commodity servers, providing high availability with no single point of failure.
```

Comment:
I think this might need to be the following to make sure that the linter doesn't complain about the lack of an oxford comma:

```suggestion
Apache Cassandra is a free, open source, distributed, wide column store, NoSQL database management system designed to handle large amounts of data across many commodity servers, providing high availability with no single point of failure.
```